pkg/cmd/cli/common/util.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
```

Comment:
@skriss how would you feel about placing these common functions in the `cli` package itself, instead of `cli/common`?

Reply:
Works for me if @shubheksha doesn't mind moving them :)

Reply:
Done 😄